### ISSUE TITLE
Icons do not show on Chrome, Firefox and IE

### ISSUE THREAD

**Issue by radrotor** [vitalidze/traccar-web#1059]
Hi.
First, fantastic development. Thank you.
Currently one issue.
The icons do not display on PC, Firefox, Chrome, IE.
They do show on my mobile.
On initial install of Traccar, icons showed on PC.
After update and traccar-web war install, icons showed on first use, then after the first login, icon disappeared.
Status popup shows data being received, time last data received, online, centers around position of the object if I move the map, but no icon.
![pc screenshot](https://cloud.githubusercontent.com/assets/25241534/22851809/003345be-f036-11e6-8f52-7d8f2e7a063a.jpg)
![mobile screenshot](https://cloud.githubusercontent.com/assets/25241534/22851810/03d123c6-f036-11e6-93e2-68ff0c579fa5.jpg)

**Comment by vitalidze** [comment]
Try changing zoom level, I recommend zooming out. There is some limitation, when icons are not showing at certain zoom level.

**Comment by radrotor** [comment]
Hi Thanks for quick reply.
Zoom is not the problem. I tried all levels.
When adding users I made the mistake of not setting expiration date, so the default expiration time on "admin" access blocked icon display
Even changing expires date afterwards did not make icons visible.
I deleted all users and re-installed same users, adding correct expiration date
Seems perfect now, displays icons at all shared user accounts
